### ISSUE TITLE
pretty printer -- map entries are no longer printed one per line on unified-builtins2

### ISSUE THREAD

**Issue by yilongli** [kframework/k-legacy#705]
This is not the case a few weeks ago. Is this the expected behavior right now? I like one entry per line much better.

**Comment by yilongli** [comment]
@dwightguth It might be related to your recent changes to `UnparserFilterNew`. Can you take a look?

**Comment by dwightguth** [comment]
This is the best we can do now because they are pretty printed as TermCons
the same as anything else. In order to improve this, it will be necessary
to implement something like Maude's format attribute. We will have to find
someone who has the time and interest to work on this if we want this to
improve. I still maintain we should find someone to primarily own the
pretty printer, because I do not have time anymore to do it myself.

tl;dr: expected behavior for now until we have more developer resources

**Comment by yilongli** [comment]
@dwightguth Did you fix this in https://github.com/kframework/k/pull/836?
